case_studies/dc2/train_dc2.sh

```diff
@@ -33,3 +33,6 @@ bliss -cp ~/bliss/case_studies/dc2 -cn config mode=train \
     train.trainer.logger.version=dc2_psf_da \
     encoder.image_normalizer.concat_psf_params=True \
     encoder.do_data_augmentation=True
+
+# new
+nohup bliss -cp ~/bliss/case_studies/dc2 -cn full_train_config > xxx.out 2>&1 &
```

Comment:
Would you modify this script so it runs the notebooks after training? e.g.,
```
jupyter nbconvert --execute DC2_galaxy_psf_params.ipynb --to html
```
Also, is there a different config file we should be using within this script, since `config.yaml` is no more? Ideally this script would run without errors a recreate the figures in the paper.